<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deleteDialog</class>
 <widget class="QDialog" name="deleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;</string>
  </property>
  <widget class="QGroupBox" name="horizontalGroupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnYesDelete">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 88, 91);
color: white;
border: none;
border-radius: 6px;
padding: 10px 20px;
font-size: 16px;
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBackDelete">
      <property name="styleSheet">
       <string notr="true">background-color: #A5D6A7;
color: #2E7D32;
border: none;
border-radius: 6px;
padding: 10px 20px;
font-size: 16px;
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(125, 92, 73);
font-size: 18px;
font-weight: bold;
border: none;
</string>
   </property>
   <property name="text">
    <string>Are you sure you want to delete this plant?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(136, 1, 3);
font-size: 16px;
border: none;
</string>
   </property>
   <property name="text">
    <string>All associated data will be permanently lost</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
